company-profiles/superplayer-and-co.md

```diff
@@ -0,0 +1,35 @@
+# Superplayer & Co
+
+## Company blurb
+
+Superplayer & Co is a leader in B2B music digital solutions in Brazil.
+We offer a full spectrum of services to help businesses innovate through music and technology. We develop custom white label solutions, background music service for businesses and trans media music content.
+Superplayer & Co was globally accelerated by Google and is part of the Movile Group, leader in mobile marketplaces in Latin America, with services like iFood, largest food delivery service in Latin America; PlayKids, world's top video streaming service focused in kids; Sympla, biggest ticketing platform for independent events in Brazil, and many others.
+
+
+## Company size
+
+1-20 (proud Superplayers).
+
+## Remote status
+
+We are a remote first company with team members spread across many cities in Brazil (also in London).
+We offer Home Office and co-working allowances for remote members like a service subscription that gives them access on many co-work spaces in the main cities in Brazil.
+Our team is from differents regions of Brazil and, as Brazil is a continental country, have a lot of cultural differences that makes our team unique!
+
+
+## Region
+
+Brazil (Brasil), Latin America.
```

Comment:
Do you hire people from outside of Brazil? Most people use the Region field to say a bit about where the company can hire from.

Reply:
Hi @nylen. Right now we hire only from Brazil and Latin America.
I added more details about it on our company profile Region section.

Thank you.